src/applications/travel-pay/components/ClaimDetailsContent.jsx

```diff
@@ -109,6 +109,19 @@ export default function ClaimDetailsContent({
               you call.
             </p>
           )}
+          {decisionLetterReason &&
+            claimsMgmtDecisionReasonToggle &&
+            (claimStatus === STATUSES.Denied.name ||
+              claimStatus === STATUSES.PartialPayment.name) && (
+              <>
+                <p className="vads-u-font-weight--bold vads-u-margin-bottom--0">
+                  {claimStatus === STATUSES.Denied.name
+                    ? 'Why we denied your claim'
+                    : 'Why we made a partial payment'}{' '}
+                </p>
+                <p className="vads-u-margin-top--0">{decisionLetterReason}</p>
+              </>
```

Comment:
I think this would be worth pulling out into a separate component:
1. If we need to style it better in the future, it will be easier to work with
2. If we need to add more logic (e.g. CFR linking), that will be easier too